examples/00-core_examples/00-basic_dpf_implementation.py

```diff
@@ -0,0 +1,217 @@
+# # ~~~~~~~ DPF PRESENTATION ~~~~~~~ # #
+
+# Ansys Data Processing Framework (DPF) provides a modular and easy to use toolbox for accessing and transforming
+# simulation data using the DPF operators.
```

Comment:
@luisaFelixSalles ok so here this example ends-up being an in-depth explanation on how to use operators in PyDPF-Core.
As we are discussing right now, maybe we will not actually put this as a basic usage example.